servers/republik/lib/profile.js

```diff
@@ -8,3 +11,10 @@ exports.isEligible = async (userId, pgdb) => {
   }))
   return hasPledges || hasMembership
 }
+
+/**
+ * Check if profile (actually user) has submitted a candidacy.
+ */
+exports.isInCandidacy = async (user, pgdb) => {
+  return (await findByUser(user, pgdb)).length > 0
```

Comment:
this would probably be worth optimising: just do a simple one field / count query.

why optimise? because this is a helper used in various places.

```
(await pgdb.public.electionCandidacies.count({ userId: user.id })).length > 0
```